ext/ffi/lib.rs

```diff
@@ -1793,13 +1988,19 @@ where
   let symbol = PtrSymbol::new(pointer, &def);
   let call_args = ffi_parse_args(scope, parameters, &def.parameters)?;
 
+  let out_buffer = out_buffer
+    .map(|v| v8::Local::<v8::TypedArray>::try_from(v.v8_value).unwrap());
```

Comment:
I don't understand. It's always going to be typed array since the out-buffer API is not user-facing but allocated by us

Reply:
It's also always a `Uint8Array`. The rest of the code always uses `ArrayBufferView` and not `TypedArray`. Asserting `Uint8Array` would make sense to make sure the code is working correctly, but `TypedArray` is kind of just an odd duck out, although more correct.

Reply:
So, there isn't anything to change?

Reply:
I would personally change the `TypedArray` usage to `ArrayBufferView` to align with the rest of the code, or to `Uint8Array` for the struct out pointers since as you said, you know it's always going to be that.